examples/end-to-end-examples/stable_diffusion/yamls/mcloud_run.yaml

```diff
@@ -10,11 +10,10 @@ compute:
 integrations:
 - integration_type: git_repo
   git_repo: mosaicml/examples
```

Comment:
can you follow up this PR with a specific commit pin for this and the other mcloud yaml?

Reply:
of course!